servicecontrol/creating-config-file.md

```diff
@@ -142,4 +142,9 @@ net stop particular.servicecontrol
 
 * `ServiceBus/AuditLogQueue` (string)
 
-  The audit queue name to use for forwarding audit messages. This only works if `ServiceControl/ForwardAuditMessages` is true. Default `<AuditQueue>.log`
\ No newline at end of file
+  The audit queue name to use for forwarding audit messages. This only works if `ServiceControl/ForwardAuditMessages` is true. Default `<AuditQueue>.log`
+
+
+* `ServiceControl/MaxBodySizeToStore` (int)
+
+  By default we only store bodies of audit messages that are smaller than 100Kb. Increase this number to store messages with larger bodies.  Default `102400`
```

Comment:
so reading this i would automatically set it to 99999999999 so i wont loose any data